lib/livebook/runtime/evaluator/doctests.ex

```diff
@@ -36,7 +36,8 @@ defmodule Livebook.Runtime.Evaluator.Doctests do
   end
 
   defp define_test_module(modules) do
-    name = Module.concat([LivebookDoctest | modules])
+    id = :erlang.phash2(modules)
```

Comment:
phash2 can definitely have conflicts. A more guaranteed way would be converting them all to strings, removing the "Elixir." prefix, joining them, and then calling :erlang.md5.